src/frontend/src/styles/globals.css

```
@@ -1689,6 +1770,127 @@ li {
+}
+
+[data-streamdown="code-block"] {
+  max-width: 100%;
+  overflow-x: auto;
+  min-width: 0;
+}
+
+.streamdown-content :where(p, li, blockquote, h1, h2, h3, h4, h5, h6) {
+  max-width: 100%;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)